.github/workflows/tests.yml

```diff
@@ -14,27 +14,27 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10.0-rc.1]
```

Comment:
FYI: `'3.10.0-rc - 3.10'` also works and ensures that GitHub Actions uses the latest version